packages/swing-store/src/swingStore.js

```diff
@@ -194,6 +194,21 @@ export function makeSwingStore(dirPath, forceReset, options = {}) {
     // { verbose: console.log },
   );
 
+  function setUnsafeFastMode(enabled) {
```

Comment:
There's probably a better name for this, but the functionality seems good to me.